### PR TITLE
Migrate engine compatibility from nodejs 6.10 to 8.10

### PR DESCRIPTION
**Pull request by arthfael-breanainn**
Hello,

AWS sended me this message through the dashboard :
> Invokes for functions configured to run on node.js 6.10 will continue to work normally, however the ability to create new Lambda functions configured to use the node.js 6.10 runtime will be disabled on April 30 2019. Code updates to existing functions using node.js 6.10 will be disabled 30 days later on May 30 2019

The migration should be anticipating. I submit these changes to your review. I hope this will be convenient.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -11,14 +10,16 @@
     "plugin:@coorpacademy/coorpacademy/prettier"
   ],
   "parserOptions": {
-    "ecmaVersion": 2016,
+    "ecmaVersion": 8,
```
Comment (AdrieanKhisbe) [inline .eslintrc]: 2017 would be better I think

**Review by AdrieanKhisbe — CHANGES_REQUESTED** [review]
Looks like you kind of mess up with the git history.

I see commit from #1.

Can you try to fix it?
If you prefer I can take care to do it, retrieving your commit and cleaning it from the other changes.
(which ean change wont happen from the PR which will be closed)

**Comment by arthfael-breanainn** [comment]
I'll try to fix my mistake with the git history.
thx for you review

**Comment by arthfael-breanainn** [comment]
I tried to fix my mistake with the git history.
Let me know if it's ok for you.

**Comment by AdrieanKhisbe** [comment]
@arthfael-breanainn There is still the merge commit: https://github.com/CoorpAcademy/aws-batch-triggering-lambda/pull/2/commits/5e6db78701aa3d1e05c4e683a3f5746c3e011f5f.

I would prefer you to rebase interactive and remove it so I can merge or rebase your PR, rather than squash it. 

sorry for the inconvenience :)

**Comment by AdrieanKhisbe** [comment]
@arthfael-breanainn Gentle reminder 😉

**Comment by arthfael-breanainn** [comment]
Thank you. I will do changes this week. sorry for the delay.

**Comment by AdrieanKhisbe** [comment]
@arthfael-breanainn No need, I made the changes: https://github.com/CoorpAcademy/aws-batch-triggering-lambda/commit/70b28226d6154aaac23c178d25e7ae4a68e23cba

Thanks you :)

**Pull request closed without merging**